start-site/src/main/resources/application.yml

```diff
@@ -1427,6 +1424,181 @@ initializr:
+          bom: spring-ai
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-ai/reference/api/clients/ollama-chat.html
+        - name: Spring AI for OpenAI
+          id: spring-ai-openai
+          compatibility-range: "[3.1.0,3.3.0-M1)"
+          group-id: org.springframework.ai
+          artifact-id: spring-ai-openai-spring-boot-starter
+          description: Support for ChatGPT, the AI language model and DALL-E, the Image generation model from OpenAI.
+          bom: spring-ai
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-ai/reference/api/clients/openai-chat.html
+        - name: Spring AI vector database for PGvector
+          id: spring-ai-vectordb-pgvector
+          compatibility-range: "[3.1.0,3.3.0-M1)"
+          group-id: org.springframework.ai
+          artifact-id: spring-ai-pgvector-store-spring-boot-starter
+          description: PGvector is an open-source extension for PostgreSQL that enables storing and searching over machine learning-generated embeddings.
+          bom: spring-ai
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-ai/reference/api/vectordbs/pgvector.html
+        - name: Spring AI vector database for Pinecone
+          id: spring-ai-vectordb-pinecone
+          compatibility-range: "[3.1.0,3.3.0-M1)"
+          group-id: org.springframework.ai
+          artifact-id: spring-ai-pinecone-store-spring-boot-starter
+          description: Pinecone is a popular cloud-based vector database, which allows you to store and search vectors efficiently.
+          bom: spring-ai
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-ai/reference/api/vectordbs/pinecone.html
+        - name: Spring AI for PostgresML
+          id: spring-ai-postgresml
+          compatibility-range: "[3.1.0,3.3.0-M1)"
+          group-id: org.springframework.ai
+          artifact-id: spring-ai-postgresml-spring-boot-starter
+          description: Support for the PostgresML text embeddings models.
+          bom: spring-ai
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-ai/reference/api/embeddings/postgresml-embeddings.html
+        - name: Spring AI vector database for Redis
+          id: spring-ai-vectordb-redis
+          compatibility-range: "[3.1.0,3.3.0-M1)"
+          group-id: org.springframework.ai
+          artifact-id: spring-ai-redis-spring-boot-starter
+          description: Redis Search and Query extends the core features of Redis OSS and allows you to use Redis as a vector database.
+          bom: spring-ai
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-ai/reference/api/vectordbs/redis.html
+        - name: Spring AI for Stability AI
```

Comment:
We need https://github.com/spring-projects/spring-ai/issues/348 to be resolved for that.